helm/defectdojo/values.yaml

```diff
@@ -20,7 +20,14 @@ createPostgresqlSecret: false
 # Option to use "postgresql" or "mysql" database type, by default "mysql" is chosen
 # Set the "enable" field to true of the database type you select (if you want to use internal database) and false of the one you don't select
 database: postgresql
+# List of allowed hosts, first entry is used for http header
```

Comment:
This comment is not correct anymore. It isn't a list. Can you please update it?

Reply:
done